.github/workflows/run-tests.yml

```diff
@@ -28,6 +28,10 @@ on:
         required: false
         type: boolean
         default: false
+      requiresTestSdk:
```

Comment:
Are these inputs documented anywhere? If not, I wonder if it would be good to just add comments on top of each briefly explaining what they are for and what do they control. Some are really obvious, but some others (like requiresTestSdk for instance) might not be super obvious.